### PR TITLE
socnetv: new port

### PR DESCRIPTION
**Pull request by szhorvat**
#### Description

New port: SocNetV

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 13.4.1 22F82 x86_64
Xcode 14.3.1 14E300c

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by szhorvat** [comment]
CC @MarcusCalhoun-Lopez 

While this compiles and works on my system, I am not confident that it will build fine in all environments. Making sure that it does would effectively require replicating what a qmake6 port group would do. Hence this request: https://trac.macports.org/ticket/67697

File: science/socnetv/Portfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           qt6 1.0
+
+name                socnetv
+github.setup        socnetv app 3.1 v
+revision            0
+
+categories          science
+platforms           darwin
```
Comment (herbygillot) [inline science/socnetv/Portfile:12]: Looks good. `platforms darwin` is now implicit in MacPorts, so this line can be dropped.

**Comment by szhorvat** [comment]
@herbygillot To be explicit about what's missing: 

 - There's a simply replacement of the `--prefix` configure argument to make it suitable for qmake, but other arguments are not handled/transformed. I don't know under what circumstances MacPorts might pass other arguments.
 - No effort is made to respect the selected compiler, or to make sure that `ccache` is actually used when selected.

All these are handled by the `qmake` (Qt 4) and `qmake5` (Qt 5) port groups. I don't have the expertise to re-implement all this for Qt 6.

File: science/socnetv/Portfile (hunk 1 above)
Comment (szhorvat) [reply science/socnetv/Portfile]: I have now removed this line.

**Comment by herbygillot** [comment]
I think that's fine for now.  I don't even think `Qt6` is compiling on every single macOS version.  This looks good as it is now, and is building successfully in CI, so we'll merge this for now.

You can commit fixes or improvements down the line as it becomes necessary to do so.

**Pull request merged**